onto-viewer-web-app/src/main/java/org/edmcouncil/spec/ontoviewer/webapp/service/LogsService.java

```diff
@@ -0,0 +1,75 @@
+package org.edmcouncil.spec.ontoviewer.webapp.service;
+
+import com.google.common.collect.Lists;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+import org.apache.commons.io.input.ReversedLinesFileReader;
+import org.apache.commons.validator.GenericValidator;
+import org.edmcouncil.spec.ontoviewer.configloader.utils.files.FileSystemManager;
+import org.edmcouncil.spec.ontoviewer.webapp.controller.LogsApiController;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LogsService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LogsApiController.class);
```

Comment:
Group fields by their types. `private static final` should be kept separated from `private final`.